pkg/operator/config/manager/manager.yaml

```diff
@@ -46,6 +46,10 @@ spec:
               value: ${API_SERVICE_SECURITY}
             - name: ALLOC_API_SVC_PORT
               value: "5000"
+            - name: MAX_NUM_GS_TO_ADD
```

Comment:
we can remove them from here so not to modify the install files. Users that want to add them can modify their Deployments manually

Reply:
Removed the variables from manager.yaml